File: laprak5/PRAK404-2210817210032-fransiskusassisiindrawijaya.c
#include <stdio.h>
int main(){
int opt;
float v1, v2, res;
while (opt != 5)
{printf("pilih program\n 1. Penjumlahan\n 2. Pengurangan\n 3. Perkalian\n 4. Pembagian\n 5. Exit\n");
printf("Masukkan Pilihan: ");
scanf("%d", &opt);
if (opt >5 || opt <1)
{printf("Input anda salah, Silahkan coba lagi\n\n");}
if (opt == 1)
{printf("Masukkan Nilai Pertama: ");scanf("%f",&v1);
printf("Masukkan Nilai Kedua: ");scanf("%f", &v2);
res = v1+v2;
printf("Hasil penjumlahan dari %.2f dengan %.2f adalah %.2f\n\n", v1,v2,res);}
else if (opt == 2)
{printf("Masukkan Nilai Pertama: ");scanf("%f",&v1);
printf("Masukkan Nilai Kedua: ");scanf("%f", &v2);
res = v1-v2;
printf("Hasil pengurangan dari %.2f dengan %.2f adalah %.2f\n\n", v1,v2,res);}
else if (opt == 3)
{printf("Masukkan Nilai Pertama: ");scanf("%f",&v1);
printf("Masukkan Nilai Kedua: ");scanf("%f", &v2);
res = v1*v2;
printf("Hasil perkalian dari %.2f dengan %.2f adalah %.2f\n\n", v1,v2,res);}
else if (opt == 4)
{printf("Masukkan Nilai Pertama: ");scanf("%f",&v1);
printf("Masukkan Nilai Kedua: ");scanf("%f", &v2);
res = v1/v2;
printf("Hasil pembagian dari %.2f dengan %.2f adalah %.2f\n\n", v1,v2,res);}
if (opt == 5 )
{printf("Terimakasih, telah menggunakan kalkulator Fransiskus Assisi indra");}}
return 0;}
    

File: laprak5/PRAK404-2210817210032-fransiskusassisiindrawijaya.py
while (True):
    print("Pilih program")
    print("1.Penjumlahan")
    print("2.Pengurangan")
    print("3.Perkalian")
    print("4.Pembagian")
    print("5.Exit")
    pil=float(input("Masukkan Pilihan :"))
    if pil==5:
        break
    elif pil>=6:
        print("Input anda salah,silahkan coba lagi\n")
        continue
    a1=float(input("Masukkan nilai pertama :"))
    a2=float(input("Masukkan nilai kedua :"))
    if pil==1:
        haspen = (a1+a2)
        print("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n"%(a1, a2, haspen)) 
    elif pil==2:
         haspen = (a1-a2) 
         print("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n"%(a1, a2, haspen))
    elif pil==3:
         haspen = (a1*a2) 
         print("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n"%(a1, a2, haspen)) 
    elif pil==4:
         haspen = (a1/a2) 
         print("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n"%(a1, a2 , haspen))
print("Terimakasih Telah Menggunakan kalkulator Fransiskus Assisi Indra Wijaya")